File: Beginner/Palindrome.cpp
#include <iostream>
#include <string>

int main() {
    std::string str = "";
    std::cin >> str;

    std:string reverse = "";
    for(int i = 0; i < str.length(); i++)
        reverse += str[str.length() - 1 - i];
    
    if(str == reverse)
        std::cout << "Palindrome!";
    else
        std::cout << "Not a palindrome!";

    return 0;
}

File: Beginner/Palindrome.java
import java.util.Scanner;

public class Palindrome {

	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		String st = s.next();
		String reverse = new StringBuffer(st).reverse().toString(); 
		if(st.equals(reverse))
			System.out.println("Palindrome");
		else
			System.out.println("Not Palindrome");
	}

}
